for editing (#11)
DevExpress-Examples/dashboard-angular-app-get-started: README.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T828585)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Dashboard for Angular - Get Started

> This project demonstrates how you can incorporate a DevExpress Dashboard component into a client-side app built with Angular. Use it as a template when you need to create a similar web application.

Reviewer: ```suggestion
This project demonstrates how to incorporate a DevExpress Dashboard component into a client-side app built with Angular. You can use it as a template when you create a similar web application.
```

> The example uses a modular client-server approach. The server (backend) project communicates with the client (frontend) application that includes all the necessary styles, scripts and HTML templates. Note that the script version on the client must match the version of libraries on the server.

Reviewer: ```suggestion
The example uses a modular client-server approach. The server (backend) project communicates with the client (frontend) application that includes all the necessary styles, scripts, and HTML templates. Note that the script version on the client should match the version of libraries on the server.
```

- The [asp-net-core-server](asp-net-core-server) folder contains the backend project built with ASP.NET Core 3.1.
- The [dashboard-angular-app](dashboard-angular-app) folder contains the client application built with Angular.

## Quick Start

> In the **asp-net-core-server** folder run the following command:

Reviewer: ```suggestion
Run the following command in the **asp-net-core-server** folder:
```

```
dotnet run
```

See the following section for information on how to install NuGet packages from the DevExpress NuGet feed: [Install DevExpress Controls Using NuGet Packages](https://docs.devexpress.com/GeneralInformation/115912/installation/install-devexpress-controls-using-nuget-packages).

> > This server allows CORS requests from _all_ origins with _any_ scheme (http or https). This default configuration is insecure: any website can make cross-origin requests to the app. We recommend that you specify the client application's URL to prohibit other clients from accessing sensitive information stored on the server. Learn more: [Cross-Origin Resource Sharing (CORS)](https://docs.devexpress.com/Dashboard/400709)

Reviewer: ```suggestion
> This server allows CORS requests from _all_ origins with _any_ scheme (http or https). This default configuration is insecure: websites can make cross-origin requests to the app. We recommend that you specify the client application's URL to prohibit other clients from accessing sensitive information stored on the server. More information: [Cross-Origin Resource Sharing (CORS)](https://docs.devexpress.com/Dashboard/400709)
```
